src/sampling.jl

```diff
@@ -396,8 +437,9 @@ Draw a random sample of `length(x)` elements from an array `a`
 and store the result in `x`. A polyalgorithm is used for sampling.
 Sampling probabilities are proportional to the weights given in `wv`,
 if provided. `replace` dictates whether sampling is performed with
-replacement and `order` dictates whether an ordered sample, also called
-a sequential sample, should be taken.
+replacement. `ordered` dictates whether
+an ordered sample (also called a sequential sample)—a sample where
+items appear in the same order as in `a`—should be taken.
```

Comment:
Parens plus dash might be a bit too much? :-)
```suggestion
an ordered sample (also called a sequential sample, i.e. a sample where
items appear in the same order as in `a`) should be taken.
```